instrument.go

```diff
@@ -54,7 +54,7 @@ func (i *Instrumenter) StreamServerInterceptor(srv interface{}, stream grpc.Serv
 func (i *Instrumenter) instrument(name string, err error, elapsed time.Duration) {
 	errtags := extractErrorTags(err)
 	status := HTTPStatusFromError(err)
-	logger := log.WithField("status", status)
+	logger := log.L().With(zap.Int("status", status)).Sugar()
```

Comment:
no need for sugar if can we also change the message format below too:

```
logger.Info("completed", zap.String("rpc", name), zap.Int("status", status), zap.Duration("elapsed", elapsed.Seconds())
```

and:
```
loggerWithTags(logger, errtags).Error("failed", zap.String("rpc", name), zap.Error(err))
```